modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java

```diff
@@ -19,10 +19,13 @@
 import com.google.common.collect.Sets;
 import com.samskivert.mustache.Mustache;
 import com.samskivert.mustache.Template;
+import io.swagger.v3.oas.models.OpenAPI;
```

Comment:
@ahmednfwela thanks for the PR. Why changed the file permission from 644 to 755 ?

Reply:
maybe was a mistake since i am using wsl2 for development, how may I fix it ?

Reply:
`chmod 644 modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java` should also work with WSL

Reply:
@kuhnroyal thanks for the help, should be solved now.